projects/js-packages/charts/src/components/pie-chart/stories/index.stories.tsx

```diff
@@ -106,54 +77,20 @@ const meta: Meta< StoryArgs > = {
 			description: 'Background color for labels displayed on pie chart segments',
 		},
 	},
-	render: ( { labelTextColor, labelBackgroundColor, theme, ...args } ) => {
-		// Create custom theme if label colors are provided
-		let customTheme: ChartTheme | undefined;
-		if ( labelTextColor || labelBackgroundColor ) {
-			let baseTheme: ChartTheme | undefined;
+	render: ( { labelTextColor, labelBackgroundColor, ...chartProps } ) => {
+		const ChartComponent = <PieChart { ...chartProps } />;
 
-			if ( typeof theme === 'object' ) {
-				baseTheme = theme as ChartTheme;
-			} else if ( theme === 'jetpack' ) {
-				baseTheme = jetpackTheme;
-			} else if ( theme === 'woo' ) {
-				baseTheme = wooTheme;
-			}
-
-			customTheme = {
-				...baseTheme,
-				labelTextColor: labelTextColor || baseTheme?.labelTextColor,
-				labelBackgroundColor: labelBackgroundColor || baseTheme?.labelBackgroundColor,
-			} as ChartTheme;
```

Comment:
Using a full theme overrides the global theme, changed to a partial theme which only overrides the label props